src/chrome/content/rules/FacebookCoreWWWi.onion.xml

```diff
@@ -1,64 +1,30 @@
-
 <!--
-
 	For other Facebook coverage, see Facebook.xml.
 
 	This ruleset covers Facebook hidden services on Tor.
 
-
-	Fully covered hosts:
-
-		- facebookcorewwwi.onion
-
-		- *.facebookcorewwwi.onion:
-
-			- (www.)?
-			- ar-ar
-			- de-de
-			- developers
-			- es-la
-			- fr-fr
-			- hi-in
-			- it-it
-			- ja-jp
-			- pt-br
-			- m
-			- zh-cn
-
-		- in *fbcdn23dssr3jqnq.onion:
-
-			- ^
-			- scontent.xx
-
+	Mismatched:
+		- fbcdn23dssr3jqnq.onion
 		- fbsbx2q4mvcl63pw.onion
-
-
-	Insecure cookies are set for these domains:
-
-		- .facebookcorewwwi.onion
-
 -->
-<ruleset name="Facebook Hidden Service">
 
-	<!--	Direct rewrites:
-				-->
+<ruleset name="Facebook Hidden Service">
 	<target host="facebookcorewwwi.onion" />
 	<target host="*.facebookcorewwwi.onion" />
 
-		<test url="http://developers.facebookcorewwwi.onion/" />
-		<test url="http://m.facebookcorewwwi.onion/" />
-		<test url="http://www.facebookcorewwwi.onion/" />
+	<target host="*.fbcdn23dssr3jqnq.onion" />
 
-	<target host="scontent.xx.fbcdn23dssr3jqnq.onion" />
+	<target host="*.fbsbx2q4mvcl63pw.onion" />
```

Comment:
We require at least 3 test urls per wildcard.

Reply:
Can you look for two more test URLs?

Reply:
I think there is no way to do this for onion domains.